builtin/mainmenu/textures.lua

```diff
@@ -145,6 +145,28 @@ function mm_texture.set_game(identifier,gamedetails)
 	
 	local path = gamedetails.path .. DIR_DELIM .."menu" ..
 									 DIR_DELIM .. identifier .. ".png"
+
+	local count = 1
+	for i = 9, 1, -1 do
+		local check_path = gamedetails.path .. DIR_DELIM .."menu" ..
+					DIR_DELIM .. identifier .. "." .. i .. ".png"
```

Comment:
I was just following the style I found in the file, which I probably shouldn't have done. How long should it be? 1 tab?